templates/zerver/footer.html

```diff
@@ -1,186 +1,108 @@
-                    {{ _("Contributing guide") }}
+                {{ _("Contributing guide") }}
                 </a>
-            </li>
-            <li><a href="/development-community/">{{ _("Development community") }}</a></li>
-            <li>
+                <a href="/development-community/">{{ _("Development community") }}</a>
                 <a href="https://zulip.readthedocs.io/en/latest/translating/translating.html">
-                    {{ _("Translation") }}
+                {{ _("Translation") }}
                 </a>
-            </li>
-            <li><a href="/api/">REST API</a></li>
-        </ul>
+                <a href="/api/">API</a>
+                <a href="https://github.com/zulip/zulip/">{{ _("GitHub") }}</a>
+            </p>
+        </div>
+        <div class="footer__section">
+            <span class="footer__section-title">
+                {{ _("ABOUT US") }}
+            </span>
+            <p>
+                <span>
+                    <a href="/team/">{{ _("Team") }}</a>
+                    &
+                    <a href="/history/">{{ _("History") }}</a>
+                </span>
+                <a href="/values/">{{ _("Values") }}</a>
+                <a href="/jobs/">{{ _("Jobs") }}</a>
+                <a href="https://blog.zulip.com/"  target="_blank">{{ _("Blog") }}</a>
+                <a href="https://twitter.com/zulip/">Twitter</a>
+                <a href="https://zulip.com/help/support-zulip-project">{{ _("Support Zulip") }}</a>
+            </p>
+        </div>
     </div>
-    <div class="footer-section">
-        <h3>{{ _("About us") }}</h3>
-        <ul>
-            <li><a href="/team/">{{ _("Team") }}</a> &amp; <a href="/history/">{{ _("History") }}</a></li>
-            <li><a href="/values/">{{ _("Values") }}</a></li>
-            <li><a href="https://twitter.com/zulip/">Twitter</a></li>
-            <li><a href="/jobs/">{{ _("Jobs") }}</a></li>
-            <li><a href="/attribution">{{ _("Website attributions") }}</a></li>
-            <li><a href="https://github.com/sponsors/zulip">{{ _("Sponsor Zulip") }}</a></li>
-        </ul>
-    </div>
-</div>
-{% else %}
-<div class="footer self-hosted-footer">
-    <div class="footer-section">
-        <ul>
-            <li>
-                <a href="/help">{{ _("Help center") }}</a>
-            </li>
-        </ul>
-    </div>
-    <div class="footer-section">
-        <ul>
-            <li>
-                <a href="/policies">{{ _("Policies") }}</a>
-            </li>
-        </ul>
-    </div>
-    <div class="break"></div>
-    <div class="footer-section powered-by-zulip">
-        <ul>
-            <li>
-                {% trans %}
-                Powered by <a href="https://zulip.com" class="coloured-text">Zulip</a>
-                {% endtrans %}
-            </li>
-        </ul>
+    {% endif %}
+    <div class="footer__legal {% if not corporate_enabled %}footer__legal_not_corporate{% endif %}">
```

Comment:
I felt like there was too much space at the bottom, with 28px bottom margin for footer section paragraphs, plus 24px top margin for this legal footer, resulting in a really tall empty space above the terms/privacy section:

![image](https://user-images.githubusercontent.com/2746074/222832804-e4c61a43-b6f4-4172-a9fc-b62fc43327a0.png)

Removing that 24px top margin results in this:

![image](https://user-images.githubusercontent.com/2746074/222832679-e40ec769-4237-4ceb-ab06-e142fe0b51c8.png)

I think that's probably what is intended (i.e. it was basically a bug that the 24px margin wasn't collapsed with the 28px margin), but @terpimost thoughts?